src/styles/NewChannel.scss

```diff
@@ -1,47 +1,45 @@
 .str-chat__new-channel {
```

Comment:
doesn't look like the styles in this file are applied anywhere so we should delete